ReactiveUI/AddTo.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace System.Reactive.Disposables
+{
+    public static class CompositeDisposableExtensions
```

Comment:
current naming policy for extension methods in RxUI is FooMixin(s) for extension methods relating to type Foo. Maybe this one (as well as the file) should be named DisposableMixin(s) ?